docs/turbo.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "./public/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "env": [
+        "SENTRY_DSN",
+        "TURBOREPO_SFDC_CAMPAIGN_ID",
+        "TRAY_URL",
+        "CONVERTKIT_API_KEY",
+        "CONVERTKIT_API_SECRET"
+      ],
+      "outputs": [".next/**"],
+      "dependsOn": ["^build", "docs#rss", "docs#schema"]
+    },
+
+    "dev": {
+      "dependsOn": ["schema"]
+    },
+    "schema": {
+      "outputs": ["public/schema.json"]
+    },
+
+    "rss": {
```

Comment:
This doesn't seem to be hooked up to anything, although a task exists. when does it run?